### PR TITLE
chore: publish v1.3.0

### PR DESCRIPTION
**Pull request by dshukertjr**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,46 @@
...
+    // Verifies a code against a challenge.
+    final res = await client.mfa
+            .verify(factorId: factorId1, challengeId: challengeId, code: getTOTP());
+    ```
+    Read more about MFA with Supabase [here](https://supabase.com/docs/guides/auth/auth-mfa)
+  - paginate `admin.listUsers()`
+    ```dart
+    auth.admin.listUsers(page: 2, perPage: 10);
+    ```
+- feat: update postgrest to v1.2.1
```
Comment (Vinzent03) [inline CHANGELOG.md:23]: Do we want to add the changes to postgrest here as well, like for the other packages here?

Comment (dshukertjr) [reply CHANGELOG.md:23]: Sorry, I should have left a note about that one, but currently long story short, I don't think anything has changed on Postgrest side of things, so not sure if we should write something there.

We have [this](https://github.com/supabase/postgrest-dart/pull/90) PR, which adds a feature to reuse isolates, but [this](https://github.com/supabase/postgrest-dart/pull/103) PR to fix the accidental breaking change makes passing isolates to PostgrestBuilder optional. Without passing any `YAJsonIsolate` instance being passed from `SupabaseClient`, calling `supabase.from()` will instantiate a new isolate every single time, resulting in the same behavior as the prior versions. 

I think the step to complete this isolate feature is
1. Add feature to accept `YAJsonIsolate` instance on functions-dart
2. Modify `SupabaseClient` to instantiate an `YAJsonIsolate` object and pass it to Postgrest and functions client.

I was going to get them done over the break, but had more things going on than I anticipated.

Comment (Vinzent03) [reply CHANGELOG.md:23]: Oh true, I've somehow forgotten there's still work to be done for that feature.

**Review by Vinzent03 — APPROVED** [review]
LGTM!

**Pull request merged**